src/applications/find-forms/sass/find-va-forms.scss

```diff
@@ -1,5 +1,9 @@
-.find-va-forms-table {
-  th {
-    min-width: 120px;
+[data-widget-type="find-va-forms"] {
+  .vads-l-grid-container--full {
+    margin-bottom: 0;
+  }
+
+  .find-va-froms-pagination-override {
+    border-top: none;
```

Comment:
This might be a naive question, but instead of doing the logic w/the search results adding a `showBorderBottom` prop, can we just override the style of the pagination's border to have the same style as the search results? -> `border-top: 1px solid $color-gray-lighter`. Then maybe we can get away without touching the JS? It's a little disappointing to need relatively complex logic to account for edge cases in the presentation, so if we can get away with just doing SCSS then that would be great.

Reply:
Wow, that's an excellent catch; I don't know how I missed that simple fix. 
Well it looks good on my end locally. I'll revert some commits and push up the pagination override.